site/_posts/2016-12-12-last-week-in-manageiq-provider-generator-pglogical-chen-primes.html.md

```diff
@@ -0,0 +1,78 @@
+The next prime is 109, meaning that 107 is called a [Chen prime](https://en.wikipedia.org/wiki/Chen_prime).
+Super cool, right? Almost as super cool as 0°! If I haven't scared you off yet,
+you can join the 45 contributors that made [255 commits][Commits merged last week] by
+checking out the [issues tracker](https://github.com/manageiq/manageiq/issues).
+
+## Improved
+
+### [pglogical Upgrade](https://github.com/ManageIQ/manageiq/pull/13070)
+pglogical had released many versions after the one we are currently using, and
+these new versions contain many bug fixes as well as the ability to fire
+triggers on replicated rows and to cleanly update a node data source name.
+However, there was a serious trigger bug that had to be resolved first. Slated
+back in July, [Nick](https://github.com/carbonin) has now been able to make the
+proper changes to be compatible and move us up to pglogical 1.2.1!
+
+### [Faster UI response on failed Git imports](https://github.com/ManageIQ/manageiq/pull/12964)
+It was found out that if whatever service in a Git import was down it would take an exceptionally long time
+for the UI to timeout and report that. Apache would eventually respond with a 502 proxy error after several minutes.
+[Erik](https://github.com/eclarizio) fixed this by removing the bottleneck, giving the client a task to check back
+on instead!
+
+## Fixed
+
+### [Authentication retry regression fixed](https://github.com/ManageIQ/manageiq/pull/13026)
+There was a slight regression following the recently merged authentication feature that retries authentication checks
+if the error is generic and unrelated to missing/invalid credentials (unreachable, etc) when running SmartState Analysis.
+[Joe](https://github.com/jrafanie) tracked it down and found the foul option that wasn't in the right place!
+
+### [Fixed save button for VM retirement date](https://github.com/ManageIQ/manageiq/pull/12343)
+The save button for VMs would not be clickable when you set a retirement date of the current day or tomorrow.
+Fixed! Now you can retire whenever you want!
+
+## New
+
+### [Load Balancers API](https://github.com/ManageIQ/manageiq/pull/13067)
+You can now retrieve load balancer information from the REST API, brought to you by [Jillian](https://github.com/jntullo)!
+
+### [New Provider generator](https://github.com/ManageIQ/manageiq/pull/12209)
+[Marcel](https://github.com/durandom) added a new Rails generator for providers! This will allow
+developers to easily create an empty scaffolding with everything you need to begin work on adding a new
+pluggable provider for ManageIQ. Just `rails generate provider YourProvider` and away you go!
+
+## Deleted
+
+### [Removed unneeded _perf_chart_flash_layout](https://github.com/ManageIQ/manageiq/pull/12989)
+Funny enough, as [Milan](https://github.com/mzazrivec) pointed out, we no longer have flash charts in the application.
+
+
```

Comment:
The two empty lines here look inconsistent, but I can't remember if that even makes a difference when this is rendered